sdk/core/Azure.Core/src/Shared/OperationInternal.cs

```diff
@@ -83,19 +83,21 @@ internal class OperationInternal : OperationInternalBase
         /// </param>
         /// <param name="scopeAttributes">The attributes to use during diagnostic scope creation.</param>
         /// <param name="fallbackStrategy"> The delay strategy to use. Default is <see cref="FixedDelayWithNoJitterStrategy"/>.</param>
+        /// <param name="requestMethod">The Http request method</param>
         public OperationInternal(IOperation operation,
             ClientDiagnostics clientDiagnostics,
-            Response rawResponse,
+            Response? rawResponse,
```

Comment:
What scenario motivates passing a null raw response?  It looks like `Operation.GetRawResponse` has a non-nullable return type.  Are you planning to return `null` from this method?  I believe that would violate the contract with the end user.

Reply:
This is the case of LRO rehydration as [here](https://github.com/Azure/azure-sdk-for-net/pull/41088/files#diff-d98a5ed3ccb11d48b0a9d4261030c3b7d7487657ad6d54ec78c6d4984c2675e7R41), the user doesn't have response in place.
And if we take a look at the documentation of `rawResponse` [here](https://github.com/Azure/azure-sdk-for-net/blob/0bcad7089e3d0690533a2261a6d9a39f3e7927dd/sdk/core/Azure.Core/src/Shared/OperationInternal.cs#L76), it says "When a user instantiates an <see cref="Operation"/> directly using a public constructor, there's no previous service call. In this case, passing <c>null</c> is expected."
So, I think it should be nullable in the first place.

Reply:
Created https://github.com/Azure/azure-sdk-for-net/pull/42666 for this change.